stubs/PyYAML/yaml/representer.pyi

```diff
@@ -1,60 +1,68 @@
+import datetime
 import sys
-from typing import Any
+from types import BuiltinFunctionType, FunctionType, ModuleType
+from typing import Any, Callable, ClassVar, Dict, List, Mapping, NoReturn, Sequence, Set, Tuple, Type, TypeVar
```

Comment:
Nit: `Callable`, `Mapping`, and `Sequence` should be imported from `collections.abc`.